### PR TITLE
Update logic of api

### PR DESCRIPTION
**Pull request by AnOldStory**

### CODE REVIEW

**Comment by CXZ7720** [comment]
I just added current timetable info field to json return, same as https://github.com/BusHanyang/ERICA_shuttlecock_API/commit/e08eedc1a77d453d9e019d21b5d30825bac6851b

File: router/routes.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+const util = require("../common");
+const fs = require("fs");
+const path = require("path");
+
+/* router */
+module.exports = function(app) {
+  /* full path */
+  app.use("/:daykind/:isWeek/:where", Validation);
```
Comment (CXZ7720) [inline router/routes.js]: 이런 신박한 방법을 왜 생각하지 못했을까;;; 😭 😭 😭

Comment (Taewan-P) [reply router/routes.js]: 흠야뤼...

**Comment by CXZ7720** [comment]
> Hmm... I couldn't fully understand how the code works, but it seems like you guys integrated the routes into "/router"

Right. This PR integrated each routers that has same code structures.
Our new router process each requests depends on `Validation`, so we can manage them with only 1 router.

**Pull request merged**